src/StarterKits/Installer.php

```diff
@@ -468,6 +469,31 @@ protected function reticulateSplines()
         return $this;
     }
 
+    /**
+     * Run any post install hooks defined in the starter kit.
+     *
+     * @return $this
+     */
+    protected function runAfterInstallHook()
+    {
+        if (! $hook = $this->config('after_install_hook')) {
+            return $this;
+        }
+
+        $hook = explode('::', $hook);
+
+        if (count($hook) != 2) {
+            return $this;
+        }
+
+        $klass = $hook[0];
```

Comment:
Is this spelling to avoid a reserved word? It's fine to use `$class`, we do it [all over the place](https://cs.github.com/statamic/cms?q=%24class).

Reply:
Just an old habit... I've been around too long :)
I'll change it.

Reply:
![](https://media.giphy.com/media/3o6wrvWxyFQ5VwdJHG/giphy-downsized.gif)